src/routes/raids/ViewRaid.tsx

```diff
@@ -29,29 +29,30 @@ const ViewRaid = () => {
       <$StatsView>
         <$Header>
           <section>
-            <h1>{data.title}</h1>
+            <h1>{data?.title}</h1>
```

Comment:
Why all the ternaries? They shouldn't need to be there anymore, as I fixed the typing specifically to avoid this!

Reply:
They all popped up as errors when I opened the file :man_shrugging: